### PR TITLE
feat:added toast

### PR DESCRIPTION
**Pull request by Snigdha17**

### CODE REVIEW

**Review by pratik-kumar-sahu — APPROVED** [review]
UI looks nice and clean go ahead with merging

File: css/components/toast.css [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+}
+
+.toast {
+  border-radius: 0.5rem;
+  max-width: 60rem;
+  width: 100%;
+  font-size: 1.8rem;
+  display: flex;
+  align-items: center;
+  justify-content: start;
```
Comment (dev-enforced) [inline css/components/toast.css]: Default value of justify-content is flex-start so you can remove it

**Review by dev-enforced — APPROVED** [review]
The code looks good and the live preview:https://deploy-preview-12--kustomize-ui.netlify.app/pages/documentation.html#toast-section 
Gave a minor suggestion which you can take care of in this branch

File: css/components/toast.css (hunk 1 above)
Comment (Snigdha17) [reply css/components/toast.css]: fixed

**Pull request merged**